fmriprep/workflows/bold/resampling.py

```diff
@@ -1268,16 +1269,18 @@ def init_bold_grayords_wf(
     from niworkflows.interfaces.utility import KeySelect
 
     workflow = Workflow(name=name)
+
+    mni_density = "2" if grayord_density == "91k" else "1"
```

Comment:
I believe the 170k grayordinates map out to ~ 1.6mm resolution
```suggestion
    mni_density = "2" if grayord_density == "91k" else "1.6"
```

Reply:
ah, i might just be misremembering. in any case, we can investigate after